packages/logger/biome.json

```diff
@@ -1,4 +1,4 @@
 {
+  "root": false,
```

Comment:
When I ran biome's migration feature to bump it, I got the same thing with all the `"root": false`. Correct me if I'm wrong, but I don't think we need it. That's what's proposed in this PR anyway https://github.com/OpenZeppelin/midnight-apps/pull/274